### ISSUE TITLE
No pokemon, gyms, or stops detected

### ISSUE THREAD

**Issue by dgordon4** [AHAAAAAAA/PokemonGo-Map#3430]
Scan is running but is detecting nothing at all. Issue started today. Niantic shut it down somehow?

## Steps to Reproduce (for bugs)

1. Run scan
2. Detect nothing
3.
4.

## Your Environment

* Version used: develop
* Environment name and version (e.g. Python 2.7):Python 2.7
* Operating System and version (desktop or mobile): Windows 10 desktop

**Comment by RobertSheaO** [comment]
Same here. Running locally.

**Comment by aldarund** [comment]
https://github.com/AHAAAAAAA/PokemonGo-Map/issues/3406

**Comment by viralmodi** [comment]
I see the same issue. Not even pokegym or pokestops which is not same as #3406

**Comment by jaller200** [comment]
From what I believe, Niantic changed their API, so the current API requests don't return the correct data

**Comment by ronchalant** [comment]
seeing the same in areas I know to be nests. wondering if niantic is soft-banning accounts?

**Comment by ChrismCruz** [comment]
Yep, Niantic changed something. See here: https://www.reddit.com/r/pokemongodev/comments/4w1cvr/pokemongo_current_api_status/

**Comment by plasticlobster** [comment]
#3426 #3416 #3411 #3410 #3408 

TL;DR: Niantic changed their API and the developers are working on a fix.

**Comment by aldarund** [comment]
@viralmodi yes it is same and your issue will be close. maybe u should look at closed issues before posting this?

**Comment by viralmodi** [comment]
@aldarund - I didn't post this issue. I just responded to a post. BTW, I had already looked at that thread and hence didn't start a separate thread. However, I was not sure if this issue and issue in that post are same since in that thread someone commented " they could see pokestops and pokegyms" - so decided to reply to this post here. Hope that clarifies things.
